scripts/build/build-utils.ps1

```diff
@@ -8,36 +8,35 @@ function Get-VsWherePath {
     return Get-ExecutablePath -name "vswhere.exe" -envVar "VSWHERE_PATH"
 }
 
-function Get-MsBuildPath([ValidateSet("14.0", "15.0")][string]$msbuildVersion) {
+function Get-MsBuildPath([ValidateSet("14.0", "15.0", "16.0", "Current")][string]$msbuildVersion) {
```

Comment:
Given the [official docs](https://docs.microsoft.com/en-us/visualstudio/msbuild/whats-new-msbuild-16-0?view=vs-2019#changed-path) say that MSBuild 16 is located under _Current_ , is there a reason to have `16.0` there?

Reply:
applies to all places where `16.0` is used

Reply:
One day, there will be 17.0 located in "Current". That's why I used it everywhere like this to follow original pattern of "14.0", "15.0".
From semantic point of view, someone may like to be explicit about version 16 (and stay there when 17 is released) and someone may just want the current, no matter what it means.

Reply:
did you test the code paths with the `16.0` folder?

I believe you should document (in comments in the script header could be a choice) a setting which involves manually creating a folder on the drive

because using the default installation will never create the `16.0` folder, right? people will have to manually create a folder and copy paste binaries?